File: 백준/Bronze/20499. Darius님 한타 안 함？/Darius님 한타 안 함？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split("/");
        int K = Integer.parseInt(input[0]);
        int D = Integer.parseInt(input[1]);
        int A = Integer.parseInt(input[2]);

        if (K + A < D || D == 0) {
            System.out.println("hasu");
        } else {
            System.out.println("gosu");
        }
    }
}

File: 백준/Bronze/20499. Darius님 한타 안 함？/Darius님 한타 안 함？.cs
  string[] input = Console.ReadLine().Split('/');
int K = int.Parse(input[0]);
int D = int.Parse(input[1]);
int A = int.Parse(input[2]);

if (K + A < D || D == 0)
{
    Console.WriteLine("hasu");
}
else
{
    Console.WriteLine("gosu");
}